contracts/orderValidation/OrderValidatorV1.sol

```diff
@@ -440,12 +447,12 @@ contract OrderValidatorV1 {
     }
 
     /**
-     * @notice Check validity of ERC-1271 maker order
-     * @param digest digest
-     * @param targetSigner the signer address to confirm message validity
-     * @param v parameter (27 or 28)
-     * @param r parameter
-     * @param s parameter
+     * @notice Check the validity of ERC-1271 maker order
+     * @param digest Digest
+     * @param targetSigner Expect signer address to confirm message validity
+     * @param v Parameter (27 or 28)
```

Comment:
in L428-430 it was documented like

`v V parameter (27 or 28)`

but here

`v Parameter (27 or 28)`

Reply:
Adjusted!